### PR TITLE
[SHIMGVW] Update Czech (cs-CZ) and Slovak (sk-SK) translation

### PR DESCRIPTION
**Pull request by Venca24**
## Purpose

Updates the Czech and Slovak translations of shimgvw.

JIRA issue: N/A

### CODE REVIEW

File: dll/win32/shimgvw/lang/cs-CZ.rc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* Translators:
  * 2008-06-24, Radek Liska aka Black_Fox (radekliska at gmail dot com)
- * 2021-02-15, Václav Zouzalík aka Venca24 (vaclav.zouzalik at seznam dot cz)
+ * 2024-05-02, Václav Zouzalík aka Venca24 (vaclav.zouzalik at seznam dot cz)
  */
```
Comment (binarymaster) [inline dll/win32/shimgvw/lang/cs-CZ.rc:4]: I'd recommend updating the copyright headers to our current standards, see for example: https://github.com/reactos/reactos/blob/master/dll/win32/shimgvw/lang/zh-TW.rc

```rc
/*
 * PROJECT:     ReactOS Picture and Fax Viewer
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0+)
 * PURPOSE:     Czech resource file
 * TRANSLATORS: Copyright 2008 Radek Liška (Black_Fox) <radekliska@gmail.com>
 *              Copyright 2021-2024 Václav Zouzalík (Venca24) <vaclav.zouzalik@seznam.cz>
 */
```

File: dll/win32/shimgvw/lang/sk-SK.rc [hunk 2]
```diff
@@ -1,5 +1,6 @@
-/* TRANSLATOR: Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk)
- * DATE OF TR: 29-05-2008
+/* Translators:
+ * 2008-05-29, Mário Kačmár aka Kario (kario at szm dot sk)
+ * 2024-05-02, Václav Zouzalík aka Venca24 (vaclav.zouzalik at seznam dot cz)
  */
```
Comment (binarymaster) [inline dll/win32/shimgvw/lang/sk-SK.rc:4]: Ditto here.

```rc
/*
 * PROJECT:     ReactOS Picture and Fax Viewer
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0+)
 * PURPOSE:     Slovak resource file
 * TRANSLATORS: Copyright 2008 Mário Kačmár <kario@szm.sk>
 *              Copyright 2024 Václav Zouzalík (Venca24) <vaclav.zouzalik@seznam.cz>
 */
```

**Review by binarymaster — COMMENTED** [review]
Hi, please configure these settings in your GitHub account, so the merging operation on PR would apply your author info correctly:

1. Set your full name in [GitHub profile settings](https://github.com/settings/profile) (it should match your git `user.name` setting)
1. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)

On the commits the author info is set correctly:

image

**Comment by Venca24** [comment]
> Hi, please configure these settings in your GitHub account, so the merging operation on PR would apply your author info correctly:
> 
>     1. Set your full name in [GitHub profile settings](https://github.com/settings/profile) (it should match your git `user.name` setting)
> 
>     2. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)
> 
> 
> On the commits the author info is set correctly:
> 
> image

Thank you for your comment, I've set it up.

**Pull request merged**